rpc/lib/client/http_client.go

```diff
@@ -83,13 +84,39 @@ func makeHTTPClient(remoteAddr string) (string, *http.Client) {
 
 //------------------------------------------------------------------------------------
 
+// JSONRPCBufferedRequest encapsulates a single buffered request, as well as its
+// anticipated response structure.
+type JSONRPCBufferedRequest struct {
```

Comment:
why expose it IF there's no access to it NOR it's an argument to some public function?

Reply:
Sure - I generally tend to err on the side of making more things publicly accessible than private, just so other devs using the API have options, but in the case of `JSONRPCBufferedRequest` specifically I suppose we could consider it a purely internal concern for now. I'll make it private 👍